ts/features/profile/components/ProfileDeleteSuccess.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { View } from "react-native";
+import {
+  ContentWrapper,
+  IOStyles,
+  FooterWithButtons,
+  Alert
+} from "@pagopa/io-app-design-system";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { useHeaderSecondLevel } from "../../../hooks/useHeaderSecondLevel";
+import I18n from "../../../i18n";
+
+type Props = {
+  onDismiss: () => void;
+};
+
+export const ProfileDeleteSuccess = ({ onDismiss }: Props) => {
+  useHeaderSecondLevel({
+    title: I18n.t("newProfile.labels.delete")
+  });
```

Comment:
When used this hook the stack navigator should be set with `screenOptions={{ gestureEnabled: isGestureEnabled, headerMode: "screen" }}` to avoid extra space on top.
![Simulator Screenshot - iPhone 15 Pro - 2024-04-24 at 21 28 31](https://github.com/gispada/io-app/assets/49342144/d8b51101-1897-4f05-b2b8-a6dda6d64e4a)

Reply:
I tried `headerMode: "screen"` but that extra space was added by the safe area.